Bootloader/firmware/src/system_config/default/system_init.c

```diff
@@ -221,20 +233,68 @@ const struct {
   uint16_t string[7];
 }
 
-sd002 = {
-  sizeof(sd002),
+PRODUCT_STRING_DESCRIPTOR = {
+  sizeof(PRODUCT_STRING_DESCRIPTOR),
   USB_DESCRIPTOR_STRING, {
     'J', 'a', ' ', 'R', 'u', 'l', 'e'
   }
 };
 
+/*
```

Comment:
It exists in both the bootloader and main application as they appear as different USB devices. I want to clean up the USB descriptor code so I may be able to merge them later.